shared/naturalcrit/markdown.js

```diff
@@ -46,6 +46,27 @@ const mathParser = new MathParser({
 		array       : false, fndef       : false
 	}
 });
+// Add absolute function
+mathParser.functions.abs = function (a) {
+	return Math.abs(a);
+};
+// Add sign function
+mathParser.functions.sign = function (a) {
+	if(a == 0) return '';
+	if(a > 0) {
+		return '\\+';
```

Comment:
Are the escapes needed?

Reply:
Yes and/or no

The reproduction text puts the `sign` and `signed` functions at the start of a line. However, the Markdown parser running after the variables and math have been resolved will quite happily accept these as list indicators, or... whatever the issue was for `+`, it escapes me right now.  
I can see it being argued either way: that these functions should output the raw text and nothing else, OR that they should output in a way that does not activate these Markdown functions.

I suspect that it comes down to "which is the better user experience?" and "which functionality is MORE unexpected?"

I started this PR leaning towards escaped output, but now I'm leaning more towards non-escaped.

Reply:
Both + and - are list markers. Based on your explanation I think escaped is better. The goal of `signed` is to get a mathematical symbol, not generate a list, so escaping would make sense then. Would be a confusing result otherwise I think.

Reply:
I've changed this PR to output the non-escaped versions now.